DBQueries.py

```diff
@@ -234,5 +237,21 @@ def select_emails_to_notify(self, users_id_list):
             logger.debug(f'Query is: {query}.')
             cur.execute(query, (users_id_list,))
             records = cur.fetchall()
-            logger.info(f"{cur.rowcount} rows affected.")
+            logger.info(f"{cur.rowcount} rows fetched.")
+        return records
+
+    def select_email_item_title_to_out_of_stock(self, items_id_list):
+        """Run SELECT query to get users mail and item id and title for each out of stock item id,
+        returning tuples with .id .title and .emails."""
+        query = '''SELECT DISTINCT i.title, string_agg(u.email, ', ') as emails FROM items as i
+                    LEFT JOIN users_items as ui on ui.item_id = i.id
+                    LEFT JOIN users as u on u.id = ui.user_id
+                    WHERE i.id in %s AND u.email IS NOT null
+                    GROUP BY i.title'''
+        self.db.get_connection()
+        with self.db.conn.cursor(cursor_factory = psycopg2.extras.NamedTupleCursor) as cur:
+            cur.execute(query, (items_id_list,))
+            logger.debug(f'Query is: {query}.')
```

Comment:
תשים את השורה הזו לפני הexecute.

אם יש בעיה בשאילתא אתה תעוף עוד לפניו

Reply:
to put the logger before the exec?